packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { FormField, FormFieldProps } from "../FormField";
+
+export interface RegionSettings {
+  countryCode?: number;
+  format?: string;
+}
+
+export interface AllowedRegions {
+  NorthAmerica: RegionSettings;
+  GreatBritain: RegionSettings;
+  Unknown: RegionSettings;
+}
+
+const REGION_SETTINGS: AllowedRegions = {
+  NorthAmerica: { countryCode: 1, format: "(###) ###-####" },
+  GreatBritain: { countryCode: 44 },
+  Unknown: {},
+};
+
+// should it be form field or common form field?
```

Comment:
Looks like `maskingProperties` is on FormField, so I think we need to use `FormFieldProps` here?

Reply:
I don't think there is one. Will remove the comment, some inputs import from that, some import from FormField.